src/pages/passengerType.tsx

```diff
@@ -37,7 +37,7 @@ const PassengerType = ({ errors = [], csrfToken }: PassengerTypeProps & CustomAp
                         </span>
                         <FormElementWrapper errors={errors} errorId={errorId} errorClass="govuk-radios--error">
                             <div className="govuk-radios">
-                                {PASSENGER_TYPES_LIST.map(
+                                {PASSENGER_TYPES_WITH_GROUP.map(
```

Comment:
Can we change the ID of these elements to be `id={`passenger-type-${passenger.passengerTypeValue}`}` rather than `id={`passenger-type-${index}`}` , for the UI tests?